Client/Frontend/Widgets/BlurTableViewController.swift

```diff
@@ -0,0 +1,182 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Storage
+
+class BlurTableViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var profile: Profile!
+    var site: Site!
```

Comment:
We shouldn't need to define these properties as explicitly unwrapped vars when they are things we can pass through the constructor. I would change these to be `let`s without the unwrap and create an `init` method that takes them.